massmigration/migrations.py

```diff
@@ -59,53 +90,62 @@ def get_backend(self):
         backend_class = import_string(self.backend_str)
         return backend_class()
 
-    def launch(self):
+    def launch(self, db_alias=None):
         """ Pass the migration to the backend to perform the data operation(s).
             This is what should be called by the web interface to trigger the migration.
         """
-        self.check_dependencies()
+        allowed_db_aliases = self.get_allowed_db_aliases()
+        if db_alias not in allowed_db_aliases:
+            raise CannotRunOnDB(
+                f"Migration {self.key} can't run on {self.database_alias}. "
```

Comment:
```suggestion
                f"Migration {self.key} can't run on {db_alias}. "
```

Apologies if this was a mistake that I introduced with one of my previous suggestions!

Reply:
Already actioned!